hrui/templates/CRM/HRUI/common/WizardHeader.tpl

```diff
@@ -0,0 +1,16 @@
+{if count( $wizard.steps ) > 1}
+  <div class="panel panel-default crm_wizard__title">
+    <div class="panel-body">
+      <ul class="nav nav-pills">
+        {section name=step loop=$wizard.steps}
+        <li class="{if $wizard.currentStepNumber == $wizard.steps[step].stepNumber}active{/if}">
```

Comment:
Is it possible to use {section name=stepN loop=$wizard.steps} `{assign var="step" value=$wizard.steps[stepN]}` and then use `step` instead of `$wizard.steps[step]`?